sql/src/main/java/org/apache/druid/sql/calcite/parser/DruidSqlReplace.java

```diff
@@ -51,22 +53,43 @@ public class DruidSqlReplace extends DruidSqlIngest
    * errors when the PARTITIONED BY custom clause is not present, and keeps its error separate from JavaCC/Calcite's
    * custom errors which can be cryptic when someone accidentally forgets to explicitly specify the PARTITIONED BY clause
    */
-  public DruidSqlReplace(
```

Comment:
please update javadocs to remove references to `partitionedByStringForUnparse`

Reply:
fixed